src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import cn from 'classnames';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partnerLabel;
+
+  if (isMarried) {
+    if (sex === 'm') {
+      partnerLabel = 'wife';
+    } else if (sex === 'f') {
+      partnerLabel = 'husband';
+    }
+  } else {
+    partnerLabel = 'not married';
+  }
```

Comment:
Here you can simplify the logic of determining the text for the partner using a Conditional (ternary) operator.

Example:

```suggestion
  if (isMarried) {
    partnerLabel = sex === MALE ? 'wife' : 'husband';
  } else {
    partnerLabel = 'not married';
  }
```